requirements/constraints.txt

```diff
@@ -254,9 +254,12 @@ pyparsing==2.2.0 \
     --hash=sha256:fee43f17a9c4087e7ed1605bd6df994c6173c1e977d7ade7b651292fab2bd010
 
 
-# Jinja2
-MarkupSafe==0.23 \
-    --hash=sha256:a4ec1aff59b95a14b45eb2e23761a0179e98319da5a7eb76b56ea8cdc7b871c3
+# Used for Jinja2
+# Code: https://github.com/pallets/markupsafe
+# Changes: https://github.com/pallets/markupsafe/blob/master/CHANGES
```

Comment:
👍 thanks for adding the project details. It appears the jump to from 0.23 to 1.0 is about the project being feature-complete, and dropping Python 2.6 support.